controllers/front/CartController.php

```diff
@@ -392,10 +392,21 @@ protected function processChangeProductInCart()
         }
 
         $product = new Product($this->id_product, true, $this->context->language->id);
+
+        $productAttributes = null;
```

Comment:
```suggestion
        $productAttributes = '';
```
Use string, you shouldn't concatenate a null value and a string.